microagents/knowledge/gitlab.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
I wonder if we need this file? It's super useful to have, but imo not yet

The resolver doesn't utilize microagents as the resolver orchestrates pulling relevant info, pushing agent changes, etc (with a token that the user provides)

In fact this [part of the file ](https://github.com/All-Hands-AI/OpenHands/pull/6458/files#diff-712ee9a1e6eb7f247083826a43571568c88e2f3663eb0ce26051d55f8093a18eR214) disables the github microagent on purpose for that reason

Maybe we can leave this for another time when we start building out the Gitlab integration in the `server` component